e2e/testdata/pod-pvc-template.yaml

```diff
@@ -29,8 +29,15 @@ metadata:
 spec:
   containers:
     - name: ubuntu
-      image: quay.io/cybozu/ubuntu:20.04
-      command: ["/usr/local/bin/pause"]
+      image: ubuntu:20.04
+      command:
+        - bash
+        - -c
+        - |
+          sleep inf &
+          trap "kill -SIGTERM $!" SIGTERM
+          wait $!
+          exit
```

Comment:
@toshipp Could you tell me why 'pause' is insufficient in this case?

Reply:
ubuntu image doesn't have the command and we can not use `registry.k8s.io/pause` image because it doesn't have tools which test needs.